### DISCUSSION TITLE
Auto Backup

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#15801] [Feature Requests]
A nice thing would be to support auto backups.

Feature wish:
- Auto create backups
- Manual create backups
- Flow support
- Import Backup
- File backup with export/import
- Scheme backup with export/import via GUI

In the meanwhile I created a plugin/extension package: https://github.com/FireboltCasters/directus-extension-auto-backup

![image](https://raw.githubusercontent.com/FireboltCasters/directus-extension-auto-backup/master/assets/autoBackupExample.png)

Feel free to test. Currently I only support SQLITE (since it was the easiest one). I think an extension for MySQL would be also pretty straight forward by using a mysql dump.

I am happy to hear about your feedback :-)

**Comment by rijkvanzanten** [comment]
Actually a "make a backup" operation would be a great extension 🙂 That way, you can just run that operation on a cron job or something

**Reply by benhaynes** [reply]
Yes to this!!

**Reply by NilsBaumgartner1994** [reply]
Oh well there is :-) just change the status to start :-)

**Reply by NilsBaumgartner1994** [reply]
It is also mentioned on the page if you install it

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
